README.md

```diff
@@ -38,8 +38,6 @@ exercises. For example, the German version of the course also uses German text
 examples and models. It's not always possible to translate all code examples, so
 some translations may still use and analyze English text as part of the course.
 
-**2.** Full translation of text examples is currently in progress.
```

Comment:
Should this line remain kept?

Reply:
Deleting that is fine since it's not used anywhere else 🙂